src/css/typography.sass

```diff
@@ -0,0 +1,165 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans&family=Poppins:wght@400;500;600;700&display=swap")
+@import 'colors'
+
+// $thin: 100
+// $extra-light: 200
+// $light: 300
+$regular: 400
+$medium: 500
+$semi-bold: 600
+$bold: 700
+// $extra-bold: 800
+// $black: 900
+// $extra-black: 950
+
+=header-globals()
+  font-family: 'Poppins', sans-serif
+
+=t-h1()
+  +header-globals()
+  font-weight: $bold
+  font-size: 56px
+  line-height: 64px
+  letter-spacing: 0.01em
```

Comment:
update this class (or "sass mixin" actually) to be in line with whatever the design / your original typography file has.

And then do the rest for all the remaining classes, that are used in the design:

- H1 ~ H5
- body 1 & 2

feel free to delete any other typography classes/mixins